packaging/install/kubernetes/README.md

```diff
@@ -5,9 +5,97 @@ Kubernetes requires that `ValidationWebhooks` are secured using TLS.
 So the web-hook service needs to have HTTPS support.
 And the CA of the certificate used for this service needs to be specified in the `ValidatingWebhookConfiguration` resource.
 
-This directory contains sample files with pre-generated certificates.
-As long as you don't change the namespace name or any service / pod names, you can just install them.
+This directory contains sample installation files without the certificates.
+You can generate the certificates by following the guide below and update the installation files with the,
 
-Additionally, in the `webhook-certificates` subdirectory, you have files which you can use to generate your own certificates using the [`cfssl` tool](https://github.com/cloudflare/cfssl).
-In case you decide to generate your own certificates, you can use the script to generate them and then you have to update the `040-Secret.yaml` and `070-ValidatingWebhookConfiguration.yaml`.
-Remember, that both resources contain the certificates encoded in base64.
\ No newline at end of file
+## Generating the certificate using OpenSSL
+
+You can use the OpenSSL utility to generate the TLS certificate for the Strimzi Drain Cleaner webhook.
+The steps below have been tested with OpenSSL 1.1.1 and should work on Linux, MacOS or in the Windows Subsystem for Linux.
+
+1) Create a subdirectory `tls-certificate` where we will generate the certificate files and enter it:
+   ```
+   mkdir tls-certificate
+   cd tls-certificate
+   ```
+2) Generate the CA public and private keys:
+   ```
+   openssl req -nodes -new -x509 -keyout ca.key -out ca.crt -subj "/CN=Strimzi Drain Cleaner CA"
+   ```
+3) Generate the private key for the Strimzi Drain Cleaner:
+   ```
+   openssl genrsa -out tls.key 2048
+   ```
```

Comment:
```suggestion
    ```
    openssl genrsa -out tls.key 2048
    ```
    
    A `tls.key` file is created.
```

Reply:
If you make the changes, check the formatting on the last couple of suggestions. They weren't working as expected in the comments boxes